src/GraphQL/transformers/mutation.js

```diff
@@ -44,13 +44,16 @@ const transformTypes = async (
             fields[field] = transformers.polygon(fields[field]);
             break;
           case inputTypeField.type === defaultGraphQLTypes.FILE_INPUT:
-            fields[field] = await transformers.file(fields[field], req);
+            // We need to use the originalFields to handle the file upload
+            // since fields are a deepcopy and do not keep the file object
```

Comment:
```suggestion
            // Use `originalFields` to handle file upload since fields are a deepcopy and do not
            // keep the file object
```